AdyenCardScanner/Sources/CardScannerAssembler.swift

```diff
@@ -33,7 +33,10 @@ class CardScannerAssembler: CardScannerAssembling {
     func resolveCardScannerViewController(
         completion: @escaping (Result<CreditCard, CardScannerError>) -> Void
     ) -> UIViewController? {
-        guard let captureDevice else { return nil }
+        guard let captureDevice else {
```

Comment:
Maybe good to expose this as a boolean to not having to try to open a viewController to get that "information"